pelux.xml

```diff
@@ -65,20 +65,20 @@
   <!-- Qt + Qt Automotive support stuff -->
   <project remote="code.qt"
            upstream="5.12"
-           revision="6830447387d2bfaef1e855040068df2a7f19d485"
+           revision="b9131d61d3f08669b493b5070811f2c617c8ee9f"
            name="yocto/meta-qt5"
            path="sources/meta-qt5"/>
 
   <project remote="code.qt"
            upstream="sumo"
-           revision="880ecb7a6e993a986d24c9a8b813bf2a48f51d24"
+           revision="c20e297c8a13b37079dc1a03fcc22d0de2bcece7"
```

Comment:
Sorry for not being clear in my previous comment, but for enhanced readability, consider use tag v5.12.1_QtAS iof hash OR add a comment about it in the commit message, e.g.
Bumped meta-boot2qt to v5.12.1_QtAS
Bumped meta-qt5 to v5.12.1
...

W/o this clarification, it's not easy at all to know which version of the projects the manifest uses.